### ISSUE TITLE
remove consensus params usage

### ISSUE THREAD

**Issue by srene** [dymensionxyz/dymint#1050]
and replace all used fields by rollapp params module

**Comment by danwt** [comment]
Hey @srene . If I understand right, you mean to remove consensus params, and add the parts which we need into rollapp params?

**Comment by srene** [comment]
yeah exactly. now we are still using some of the cosmos consensus params like maxgas, but ignoring others. the idea is to have all relevant params in a signle module (rollapp params)
